airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -472,3 +515,189 @@ class Labels(GoogleAdsStream):
     """
 
     primary_key = ["label.id"]
+
+
+class ChangeStatus(IncrementalGoogleAdsStream):
+    """
+    Change status stream: https://developers.google.com/google-ads/api/fields/v14/change_status
+    Stream is only used internally to implement incremental updates for child streams of IncrementalEventsStream
+    """
+
+    cursor_field = "change_status.last_change_date_time"
+    slice_duration = pendulum.duration(days=15)
```

Comment:
Is there a reason for having a 14-day period for `IncrementalGoogleAdsStream ` and replace it with 15-day period here?

Reply:
Fixed